File: data/src/main/java/ru/timekeeper/data/network/model/groupWallRemote/Group.java
package ru.timekeeper.data.network.model.groupWallRemote;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Group {

    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("screen_name")
    @Expose
    public String screenName;
    @SerializedName("is_closed")
    @Expose
    public Integer isClosed;
    @SerializedName("type")
    @Expose
    public String type;
    @SerializedName("is_admin")
    @Expose
    public Integer isAdmin;
    @SerializedName("is_member")
    @Expose
    public Integer isMember;
    @SerializedName("is_advertiser")
    @Expose
    public Integer isAdvertiser;
    @SerializedName("photo_50")
    @Expose
    public String photo50;
    @SerializedName("photo_100")
    @Expose
    public String photo100;
    @SerializedName("photo_200")
    @Expose
    public String photo200;

}

File: data/src/main/java/ru/timekeeper/data/network/model/groupsRemote/Group.kt
package ru.timekeeper.data.network.model.groupsRemote

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Group(

        @SerializedName("id")
        @Expose
        val id: Int? = null,
        @SerializedName("name")
        @Expose
        val name: String? = null,
        @SerializedName("screen_name")
        @Expose
        val screenName: String? = null,
        @SerializedName("is_closed")
        @Expose
        val isClosed: Int? = null,
        @SerializedName("type")
        @Expose
        val type: String? = null,
        @SerializedName("is_admin")
        @Expose
        val isAdmin: Int? = null,
        @SerializedName("is_member")
        @Expose
        val isMember: Int? = null,
        @SerializedName("is_advertiser")
        @Expose
        val isAdvertiser: Int? = null,
        @SerializedName("photo_50")
        @Expose
        val photo50: String? = null,
        @SerializedName("photo_100")
        @Expose
        val photo100: String? = null,
        @SerializedName("photo_200")
        @Expose
        val photo200: String? = null,

        var isFavorite: Boolean = false

)
